src/clj/game/cards/upgrades.clj

```diff
@@ -726,18 +727,17 @@
 (defcard "Increased Drop Rates"
   {:flags {:rd-reveal (req true)}
    :access {:interactive (req true)
-            :optional
-            {:player :runner
-             :waiting-prompt "Runner to choose an option"
-             :prompt "Take 1 tag to prevent Corp from removing 1 bad publicity?"
-             :yes-ability
-             {:async true
-              :effect (effect (system-msg "takes 1 tag to prevent Corp from removing 1 bad publicity")
-                              (gain-tags eid 1 {:unpreventable true}))}
-             :no-ability
-             {:msg "remove 1 bad publicity"
-              :effect (effect (lose-bad-publicity :corp 1)
-                              (effect-completed eid))}}}})
+            :player :runner
+            :async true
+            :waiting-prompt "Runner to choose an option"
+            :msg (req (if (= target "The Corp removes 1 bad publicity")
+                       "to remove 1 bad publicity"
+                       (msg "force the Runner to " (decapitalize target))))
+            :prompt "Choose one"
+            :choices ["Take 1 tag" "The Corp removes 1 bad publicity"]
```

Comment:
Can you choose removing bad pub if corp has none?

Reply:
[Yes](https://netrunnerdb.com/en/card/26054), since it's not a cost.

Reply:
Cool, thanks.